docs/builtin-functions.md

```diff
@@ -78,7 +78,7 @@ Python contains a limited number of builtin functions defined [here](https://doc
 
 | Method | Supported |
 |----------|-----------|
-| `append` | Python-only |
+| `append` | Python and C only |
```

Comment:
Do you think we could add Fortran to our would you rather do that in a separate PR?

Reply:
Done!

Reply:
Great! Thanks :)